openbb_terminal/common/behavioural_analysis/twitter_view.py

```diff
@@ -152,52 +153,111 @@ def display_sentiment(
     df_tweets = df_tweets.sort_values(by="date")
     df_tweets["cumulative_compound"] = df_tweets["sentiment"].cumsum()
 
-    # This plot has 2 axis
-    if external_axes is None:
-        _, axes = plt.subplots(
-            2, 1, sharex=True, figsize=plot_autoscale(), dpi=cfg_plot.PLOT_DPI
+    # get stock end price for each corresponding day if compare == True
+    if not compare:
+        # This plot has 2 axis
+        if external_axes is None:
+            _, axes = plt.subplots(
+                2, 1, sharex=True, figsize=plot_autoscale(), dpi=cfg_plot.PLOT_DPI
+            )
+            ax1, ax2 = axes
+        else:
+            if len(external_axes) != 2:
+                logger.error("Expected list of one axis item.")
+                console.print("[red]Expected list of one axis item./n[/red]")
+                return
+            (ax1, ax2) = external_axes
+
+        ax1.plot(
+            pd.to_datetime(df_tweets["created_at"]),
+            df_tweets["cumulative_compound"].values,
+        )
+        ax1.set_ylabel("\nCumulative\nVADER Sentiment")
+        for _, day_df in df_tweets.groupby(by="Day"):
+            day_df["time"] = pd.to_datetime(day_df["created_at"])
+            day_df = day_df.sort_values(by="time")
+            ax1.plot(
+                day_df["time"],
+                day_df["sentiment"].cumsum(),
+                label=pd.to_datetime(day_df["date"]).iloc[0].strftime("%Y-%m-%d"),
+            )
+            ax2.bar(
+                df_tweets["date"],
+                df_tweets["positive"],
+                color=theme.up_color,
+                width=theme.volume_bar_width / 100,
+            )
+        ax2.bar(
+            df_tweets["date"],
+            -1 * df_tweets["negative"],
+            color=theme.down_color,
+            width=theme.volume_bar_width / 100,
+        )
+        ax1.set_title(
+            f"Twitter's {ticker} total compound sentiment over time is {round(np.sum(df_tweets['sentiment']), 2)}"
         )
-        ax1, ax2 = axes
+
+        theme.style_primary_axis(ax1)
+
+        ax2.set_ylabel("VADER Polarity Scores")
+        theme.style_primary_axis(ax2)
+
     else:
-        if len(external_axes) != 2:
-            logger.error("Expected list of one axis item.")
-            console.print("[red]Expected list of one axis item./n[/red]")
-            return
-        (ax1, ax2) = external_axes
+        closing_price_df = get_closing_price(ticker, n_days_past - 1)
+        # This plot has 3 axis
+        if external_axes is None:
+            _, axes = plt.subplots(
+                3, 1, sharex=False, figsize=(12, 7), dpi=cfg_plot.PLOT_DPI
```

Comment:
Here the fig size should be `plot_autoscale()`

Reply:
I see, I tried this, but then with three figs, the individual plots are quiet small. Wouldn't that be a problem?

Reply:
That may be because our default is not ideal, but this should be able to be configurable by the user